examples/nlp/bert_pretraining.py

```diff
@@ -118,6 +118,8 @@
                         Used for preprocessed data")
 parser.add_argument("--dataset_name", default="wikitext-2", type=str)
 parser.add_argument("--load_dir", default=None, type=str)
+parser.add_argument("--bert_checkpoint", default=None, type=str,
+                    help="specify path to pretrained BERT weights")
```

Comment:
you mean file path to module checkpoint instead of just checkpoint_dir? if so, we should probably clarifiy